### DISCUSSION TITLE
How about adding support for Weather or Seasons?

### DISCUSSION THREAD

**Discussion by brkeejp** [t1m0thyj/WinDynamicDesktop#291] [Ideas]
I really like to see Weather and/or seasons support. I know its not something easy to do but just suggestion.

I can even try to create BG pack for this.

**Comment by panar85** [comment]
I'm searching for weather based dynamic wallpapers..That would be a great idea!

**Comment by t1m0thyj** [comment]
This would be cool, but is not a planned feature for WDD since it would require creating a entirely new wallpaper scheduler (see https://github.com/t1m0thyj/WinDynamicDesktop/issues/35).

**Reply by brkeejp** [reply]
thanks for information.
